llvm/test/CodeGen/SPIRV/hlsl-intrinsics/splitdouble_vector.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: opt -S -scalarizer -mtriple=spirv-vulkan-library %s 2>&1 | llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown -o - | FileCheck %s
+
+; SPIRV lowering for splitdouble should relly on the scalarizer.
```

Comment:
This comment doesn't sound right. We don't currently use the scalarizer when targeting SPIRV. I don't understand why we would start for the split double case.